packages/repository/src/entity/factory.test.ts

```diff
@@ -146,13 +166,39 @@ describe("Entity", () => {
 
         expect(entity.isSynced(firstSyncKey)).toBe(false)
       })
+
+      it("Given schema, methods, When entity factory called, Then return entity with methods", ({
+        entitySchema,
+      }) => {
+        const testMethod = vi.fn()
+
+        const entityFactory = entityModelFactory({
+          schema: entitySchema,
+          methods: {
+            testMethod,
+          },
+        })
+        const data = {
+          foo: "bar",
+        }
+
+        const entity = entityFactory.createEntity(data)
+
+        expect(entity.testMethod).toBeDefined()
+        expect(entity.testMethod).toBeTypeOf("function")
+
+        entity.testMethod()
+
+        expect(testMethod).toHaveBeenCalled()
+      })
```

Comment:
Czy ten test case title jest dobry? czytając go na końcu się spodziewałem `   expect(result).toEqual(
          expect.objectContaining({
            createEntity: expect.any(Function),
            recoverEntity: expect.any(Function),
          })
        )`

Reply:
BTW czemu nie korzystasz z Gherkina i nie numerujesz testów? jak się któryś wysypie to ciężko znaleźć :D